examples/ruby/README.md

```diff
@@ -9,38 +9,38 @@ Using official Looker Ruby SDK: https://github.com/looker/looker-sdk-ruby/
 Use cases with Looker API:
 ___________
 ### USERS 👤
-- Logout all users on the instance [[link]](logout_all_users.rb)
-- Disable users in the instance [[link]](disable_users.rb)
-- Get a list of all users and their auth credentials [[link]](users_list_and_auth_types.rb)
-- Add all users to a group [[link]](all_users_to_group.rb)
+- Logout all users on the instance [[link]](https://github.com/looker-open-source/sdk-examples/blob/master/ruby/logout_all_users.rb)
+- Disable users in the instance [[link]](https://github.com/looker-open-source/sdk-examples/blob/master/ruby/disable_users.rb)
+- Get a list of all users and their auth credentials [[link]](https://github.com/looker-open-source/sdk-examples/blob/master/ruby/users_list_and_auth_types.rb)
+- Add all users to a group [[link]](https://github.com/looker-open-source/sdk-examples/blob/master/ruby/all_users_to_group.rb)
```

Comment:
Are these links not resolving when left relative? Fully qualifying the URL shouldn't be required, and isn't the right pattern, IMO.

Reply:
Updated the README with relative links now. Thanks for the feedback John!